src/opf/ptdf.jl

```diff
@@ -0,0 +1,112 @@
+using SparseArrays
+using PowerModels
+using LinearAlgebra
+
+abstract type AbstractPTDF end
+
+
+struct LazyPTDF{TF} <: AbstractPTDF
```

Comment:
Type parametrization no longer needed
```suggestion
struct LazyPTDF <: AbstractPTDF
```